app/services/start_check_service.rb

```diff
@@ -34,18 +39,13 @@ def perform
   private
 
   def process_submissions
-    successes = []
-    failures = []
-
+    results = []
     submissions = check.submissions
     submissions.each do |submission|
       zip = deserialize_submission_zip(submission)
-      successes << process_submission(zip, submission)
-    rescue StandardError => e
```

Comment:
this was an issue. we only want to suppress the babel stripper related errors. any other errors are possibly bugs.